scripts/check_frontend_test_coverage.py

```diff
@@ -0,0 +1,407 @@
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Check for decrease in coverage from 100% of frontend files."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import os
+import re
+import shutil
+import subprocess
+import sys
+
+import python_utils
+
+LCOV_FILE_PATH = os.path.join(os.pardir, 'karma_coverage_reports', 'lcov.info')
+PR_LCOV_FILE_PATH = os.path.join(os.curdir, 'tmp', 'pr-lcov.info')
```

Comment:
I just had an idea. What if you just store pr-lcov.info in the karma coverage reports folder instead? I think that makes more sense than modifying the user's filesystem.

Ideally maybe make the filename pr-lcov-{{some kind of date/timestamp}}.info so that things don't get automatically rewritten.

Reply:
I decided to take off the PR_LCOV_FILE_PATH, since the diff of lcov files won't happen anymore. Now just lcov.info from karma_coverage_reports is being used.